andela/ah-infinity-stones-frontend
# #161966876 Implement Article CRUD

#### What does this PR do?
- Implement article CRUD

#### Description of Task to be completed?
- create article creation component
- create view for seeing all articles
- create view for seeing a single article
- create article update component
- enable deletion of an article

#### How should this be manually tested?
1. `$ git clone -b ft-users-create-articles-161966876 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  Open http://localhost:3000/  to view it in the browser. To create an article click on the article link in the navbar. Click any of the articles and you should see the particular article. You can also click on edit or delete button to edit the article or delete it respectively.
The page will reload if you make edits. 	You will also see any lint errors in the console.
4. Test the app by running `$ npm test` - This will Launch the test runner in the interactive watch mode.

#### What are the relevant pivotal tracker stories?
[#161966876](hhttps://www.pivotaltracker.com/story/show/161966876)

#### Screenshots
<img width="1410" alt="screenshot 2019-01-21 at 09 59 28" src="https://user-images.githubusercontent.com/16904946/51457926-54e8f880-1d64-11e9-90d4-f535035cade1.png">
<img width="1414" alt="screenshot 2019-01-21 at 09 59 40" src="https://user-images.githubusercontent.com/16904946/51457929-587c7f80-1d64-11e9-8b94-64f9bac46d55.png">
<img width="1408" alt="screenshot 2019-01-21 at 10 04 25" src="https://user-images.githubusercontent.com/16904946/51457939-67633200-1d64-11e9-8c52-7fa08b6a4254.png">
<img width="1412" alt="screenshot 2019-01-21 at 10 04 38" src="https://user-images.githubusercontent.com/16904946/51457942-6c27e600-1d64-11e9-9b09-e64ac4139cf5.png">
<img width="1410" alt="screenshot 2019-01-21 at 10 03 12" src="https://user-images.githubusercontent.com/16904946/51457952-73e78a80-1d64-11e9-80fc-76a52c9ef9a1.png">
<img width="1410" alt="screenshot 2019-01-21 at 10 03 29" src="https://user-images.githubusercontent.com/16904946/51457957-777b1180-1d64-11e9-935c-a0d9549cebc6.png">
<img width="1418" alt="screenshot 2019-01-21 at 10 06 01" src="https://user-images.githubusercontent.com/16904946/51457964-7ea21f80-1d64-11e9-9711-5510670e0674.png">
<img width="1414" alt="screenshot 2019-01-21 at 10 06 19" src="https://user-images.githubusercontent.com/16904946/51457973-8366d380-1d64-11e9-89f5-21d0ed731c44.png">
<img width="1418" alt="screenshot 2019-01-21 at 10 06 31" src="https://user-images.githubusercontent.com/16904946/51457977-8661c400-1d64-11e9-8058-b061c74b84d8.png">
